webui/src/components/OrgTree.vue

```diff
@@ -8,9 +8,10 @@
                 <span class="org-tree-item-title">{{user.officeLocation}}</span>
                 <span class="org-tree-item-links">
                     View in
-                    <a class="org-tree-item-link" target="_blank" :href="user.delvelink">Delve</a>
-                    <a class="org-tree-item-link" target="_blank" :href="user.o365link">O365</a>
```

Comment:
And what about this link?

Reply:
Since this link is available only for Office365 administrators and is useless for most of the AAD users I decided to remove the link.